assets/scss/base/_colours.scss

```diff
@@ -40,3 +40,12 @@ $modal-dialog-yellow: $attorney-yellow;
 
 // NOTE this is a grey from govuk_elements, which will be removed, so adding a version of it here
 $govuk-grey-3: #DEE0E2;
+
+$attorney-border-left-client-request-accepted: #006435;
+$attorney-bg-client-request-accepted: #BFD8CD;
+
+$attorney-border-left-client-request-requested: #F47738;
+$attorney-bg-client-request-requested: #FCDDCD;
+
+$attorney-border-left-client-request-rejected: #B10E1E;
+$attorney-bg-client-request-rejected: #ECC3C7;
```

Comment:
Naming wise they can be a little cleaner as please separate colours and colour functions so we can reuse these colours for other things as per http://sachagreif.com/sass-color-variables

```
//colour
$lime-green: #006435;
$light-lime-green: #BFD8CD;


// colour function
$client-request-accepted-bg: $lime-green;
$client-request-accepted-border: $light-lime-green;
```